add config with prefix

ta-tikoma opened the pull request.

ta-tikoma committed:
add config with prefix

VincentBean commented on config/horizon-dashboard.php:
```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'prefix' => env('HORIZON_DASHBOARD_PREFIX', 'horizon-dashboard'),
```
I do not think that the path should change per environment so please remove the `env`

VincentBean commented on src/ServiceProvider.php:
```diff
@@ -71,6 +71,7 @@ protected function registerCommands(): static
     public function boot()
```
Please add this to the register function above so that users do not have to publish the config:

```php
$this->mergeConfigFrom(__DIR__.'/../config/horizon-dashboard.php', 'horizon-dashboard');
```

ta-tikoma committed:
postgresql error date size

`SQLSTATE[22008]: Datetime field overflow: 7 ERROR:  date/time field value out of range: "1693562723"
HINT:  Perhaps you need a different "datestyle" setting. (SQL: select avg("runtime") as aggregate from "job_statistics" where "queue" = default and "finished_at"::date > 2023-09-01)`

ta-tikoma committed:
remove env and add config merge

ta-tikoma committed:
convert date to unix format
